docs/source/contributing.rst

```diff
@@ -0,0 +1,187 @@
+Contributing to Kordac
+#######################################
+
+The git repository for Kordac can be found here_.
+
+.. _here: https://github.com/uccser/kordac
+
+
+Issue Reporting and Bug Fixes
+=======================================
+If you come across a bug in Kordac, please `report it`_ on the repo.
+
+<Bug fix? Add tests.>
+
+.. _report it: https://github.com/uccser/kordac/issues
+
+The Code Base
+=======================================
+If you would like to contribute to Kordac, fork the repository.
+
+Overview
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Before reading this section, make sure you have read `how to use Kordac`_ (or even better, have already used Kordac!).
+
+.. _how to use Kordac: http://kordac.readthedocs.io/en/develop/usage.html
+
+
+There are several parts and terms of Kordac to become familiar with:
+
+- **Tag**
+    This refers to the custom markdown syntax that Kordac processes.
+    
+    For example:
+    
+    .. code-block:: none
+
+      {comment this will be removed by the converter}
+
+      {image file-path='img/totally-real-image.png' alt='process me'}
+    
+    are examples of the ``comment`` and ``image`` tags in Kordac.  
+
+- ``Kordac()``
+	The convertor object itself. This is what a user will use to create a Kordac converter, and what is used to define a custom processor list, custom html templates and custom Markdown Extensions to use.
+
+- ``KordacResult()``
+    The object returned by ``Kordac()`` containing:
+    	- Converted html string
+    	- Title
+    	- Required files (images, interactives, scratch images, page scripts)
+    	- Heading tree
+    	- Required glossary terms
+
+- ``KordacExtension()``
```

Comment:
Probably want to describe the relationship between `Kordac`, `KordacExtension`, and `markdown` in more detail, possibly with a diagram. It would be confusing to understand why Kordac isn't just an extension. Explain why it's also a markdown wrapper.